src/screens/nano/BlueprintList.js

```diff
@@ -123,7 +123,6 @@ function BlueprintList() {
             // so we must clear the id and page queryParams
             pagination.clearParametersWithoutRefresh(['id', 'page']);
           }
-          return;
```

Comment:
What is this?

Reply:
Same here, removed because of the [`no-useless-return`](https://eslint.org/docs/latest/rules/no-useless-return) lint rule.